dist/toolwindow.js

```diff
@@ -401,6 +419,9 @@ function _defineProperty(obj, key, value) { if (key in obj) { Object.definePrope
       refresh: function refresh() {
         var _this3 = this;
 
+        if (this._options.title && this._options.title != "") { //## Update title
```

Comment:
minor: empty string is falsey, so this can be simplified to:
```
if (this._options.title) {
	_this3._setTitle(this._options.title);
}
```

Also, `_this3` is quite an odd name for a variable -- I know it existed there before (probably from the code I started with), but feel free to rename to something more recognisable, like `self`

Reply:
done